composer.json

```diff
@@ -21,8 +24,9 @@
     "symfony/form": "^6.2"
   },
   "require-dev": {
-    "phpstan/phpstan": "^1.10.25",
-    "phpstan/phpstan-symfony": "^1.2.14",
+    "phpunit/phpunit": "^9.5",
```

Comment:
@aryaantony92 This is the important line. Otherwise it will update zu version 10. This line is also in the other pimcore packages. The gitignore have old lines and doesnt ignore generated test files

Reply:
@aryaantony92 With the new codeception version 5.0.11 it works with phpunit 10. I revert this and bump the codeception version:
https://github.com/Codeception/Codeception/releases/tag/5.0.11
https://github.com/pimcore/ecommerce-framework-bundle/pull/117/commits/1a38e9e76a817ac1cf366d686763f1926ebd9c5b